modules/installation-guide/partials/proc_importing-untrusted-tls-certificates.adoc

```diff
@@ -149,42 +121,27 @@ $ helm upgrade che -n che --set global.tls.serverTrustStoreConfigMapName=<config
 ----
 +
 When using Minikube to run {prod-short}, substitute _<kubernetes-cluster-domain>_ with `$(minikube ip).nip.io`.
-====
-endif::[]
 
-. Restart the {prod-short} Operator, the {prod-short} server and {identity-provider} to load the new certificates:
-+
-[subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
-----
-$ kubectl rollout restart -n __<{project-context}-namespace-name>__ deployment/{prod-id-short}-operator
-$ kubectl rollout restart -n __<{project-context}-namespace-name>__ deployment/{prod-id-short}/keycloak
-$ kubectl rollout restart -n __<{project-context}-namespace-name>__ deployment/{prod-id-short}
-----
-+
-[NOTE]
-====
-Restarting the {prod-short} components is not required anymore on {prod-short}
-ifeval::["{project-context}" == "che"]
-v7.20.0
 endif::[]
-ifeval::["{project-context}" != "che"]
-2.5.0
-endif::[]
-and higher.
-====
 
 == Verification at the {prod-short} installation level
 
-If you added the certificates without error, the {prod-short} server starts and obtains {identity-provider} configuration over https. Otherwise here is a list of things to verify: 
+If after adding the certificates something does not work as expected, here is a list of things to verify:
 
-- In case of a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operator] deployment, the `CheCluster` attribute `serverTrustStoreConfigMapName` value matches the name of the ConfigMap. Get the value using the following command :
+- In case of a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operator] deployment, namespace where `CheCluster` located contains labeled config maps with right content:
 +
 [subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
 ----
-$ {orch-cli} get -o json checluster/{prod-checluster} -n __<{project-context}-namespace-name>__ | jq .spec.server.serverTrustStoreConfigMapName
+$ {orch-cli} get cm --selector=app.kubernetes.io/component=ca-bundle,app.kubernetes.io/part-of=che.eclipse.org -n __<{project-context}-namespace-name>__
```

Comment:
We should mention `spec.server.serverTrustStoreConfigMapName` as well for backward compatibility

Reply:
I created a link to a separate page that contains old version of this page.